tests/unit/processing/test_process_download.py

```diff
@@ -57,6 +57,18 @@ def test_download_from_s3_uri(self):
             bucket_name="fake_bucket_name", object_key="fake_key/fake_file.h5ad", local_filename="fake_local_path"
         )
 
+    def test_download_from_dropbox_uri(self):
+        """
+        Call process download using a dropbox uri
+        """
+
+        dropbox_uri = "https://www.dropbox.com/s/fake_location/test.h5ad?dl=1"
+        pdv = ProcessDownload(self.business_logic, self.uri_provider, self.s3_provider)
+        pdv.download = Mock()
+
+        assert pdv.download_from_source_uri(dropbox_uri, "fake_local_path") == "fake_local_path"
+        pdv.download.assert_called_once_with(dropbox_uri, "fake_local_path")
+
```

Comment:
Perhaps you added this test intentionally, separate from my comment on this file, but what I had meant to request was a test that checks for [the error raised on ln 70 of process_download.py](https://github.com/chanzuckerberg/single-cell-data-portal/blob/bbebe52ef2a369662f54eab917ecb2b8a602c7bb/backend/layers/processing/process_download.py#L70), which is why I suggested `http` since it is not an accepted URI scheme for our ingestion pipeline

Reply:
That line isn't actually reachable. Everything that fails will fail at line 49. I plan to remove most of this logic in a refactor anyways.

Reply:
the refactor https://github.com/chanzuckerberg/single-cell-data-portal/pull/6129